docs/quick.md

```diff
@@ -75,7 +78,7 @@ $ pull_results
 $ plot_final_task_accuracies --group_name=GROUP_NAME
```

Comment:
This script is actually removed and its now only:

```shell
aggregate --group_name GROUP_NAME
```

Reply:
Will be fixed once I rebase #112